src/main/java/com/atlassian/plugins/confluence/markdown/ext/DevOpsResizableImage/internal/ResizableImageInlineParserExtension.java

```diff
@@ -0,0 +1,82 @@
+package com.atlassian.plugins.confluence.markdown.ext.DevOpsResizableImage.internal;
+
+import com.atlassian.plugins.confluence.markdown.ext.DevOpsResizableImage.ResizableImage;
+import com.vladsch.flexmark.parser.InlineParser;
+import com.vladsch.flexmark.parser.InlineParserExtension;
+import com.vladsch.flexmark.parser.InlineParserExtensionFactory;
+import com.vladsch.flexmark.parser.LightInlineParser;
+import com.vladsch.flexmark.util.sequence.BasedSequence;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Set;
+import java.util.regex.Pattern;
+
+public class ResizableImageInlineParserExtension implements InlineParserExtension {
+    final public static Pattern IMAGE_PATTERN = Pattern.compile("\\!\\[(\\S*)\\]\\((\\S*)\\s*=*(\\d*)x*(\\d*)\\)",
+            Pattern.CASE_INSENSITIVE);
+
+    public ResizableImageInlineParserExtension(LightInlineParser inlineParser) {
+    }
+
+    @Override
+    public void finalizeDocument(@NotNull InlineParser inlineParser) {
+    }
+
+    @Override
+    public void finalizeBlock(@NotNull InlineParser inlineParser) {
+    }
+
+    @Override
+    public boolean parse(@NotNull LightInlineParser inlineParser) {
+        int index = inlineParser.getIndex();
+        char c = inlineParser.getInput().charAt(index + 1);
```

Comment:
Doesn't this mean that this will only work at the start of a line? Can this be used inline?

Reply:
Yes, it sure works inline as well. That\`s why inlineParser is used. I\`ve tested that.